templates/assembly-script/.github/workflows/release.hippo.yml

```
@@ -38,22 +38,22 @@ jobs:
         npm install
         npm run build
 
-    - name: Install hippofactory
+    - name: Install hippo CLI
       run: |
         mkdir -p tools/tmp
-        wget https://github.com/deislabs/hippofactory/releases/download/v0.5.0/hippofactory-v0.5.0-linux-amd64.tar.gz -q -O ./tools/tmp/hippofactory.tar.gz
-        tar xf ./tools/tmp/hippofactory.tar.gz -C ./tools
+        wget https://github.com/deislabs/hippo-cli/releases/download/v0.6.0/hippo-v0.6.0-linux-amd64.tar.gz -q -O ./tools/tmp/hippo.tar.gz
+        tar xf ./tools/tmp/hippo.tar.gz -C ./tools
 
     - name: Publish bindle (tagged)
       if: startsWith(github.ref, 'refs/tags/v')
       env:
         HIPPO_USERNAME: ${{ secrets.HIPPO_USERNAME }}
         HIPPO_PASSWORD: ${{ secrets.HIPPO_PASSWORD }}
-      run: ./tools/hippofactory . -v production -o message
+      run: ./tools/hippo . -v production -o message
```

This and elsewhere need to be `hippo push`: 

```suggestion
      run: ./tools/hippo push . -v production -o message
```